world/petitions/petitions_commands.py

```diff
@@ -8,9 +8,179 @@
 from server.utils.prettytable import PrettyTable
 from world.petitions.forms import PetitionForm
 from world.petitions.exceptions import PetitionError
-from world.petitions.models import BrokeredSale, Petition
+from world.petitions.models import BrokeredSale, Petition,WantedAd
+from web.character.models import (Clue, SearchTag)
 
 
+
+
+class CmdMatchmaker(ArxCommand):
+    """
+    Creates or searches wanted ads
+    
+        Usage:
+    -Viewing:
+        matchmaker
+        matchmaker/all
+    -Editing
+        matchmaker/active #Toggles on/off
+        matchmaker/blurb <text>
+        matchmaker/playtimes <start>,<end> #Assume EST
+        matchmaker/clues <clue1>,<clue2>
+        matchmaker/tags <tag1>,<tag2>
+        matchmaker/seeking <marriage>,<plot>,<protege>,<sponsor> #any
```

Comment:
Some brief examples of use for the docstring would probably be a good idea here.